src/main/java/pw/itr0/kaba/encrypt/PBE.java

```diff
@@ -234,16 +231,13 @@ private Decrypter(byte[] salt, int iterationCount) {
         }
 
         /**
-         * Decrypt secret bytes using password for PBE.
-         *
-         * @param password decrypting password
-         * @param secret   secret bytes
-         * @return PBE decrypted secret
-         * @throws GeneralSecurityException on decryption error
+         * {@inheritDoc}
+         * <p>
+         * This implementation return PBE decryption {@link Cipher}.
          */
         @Override
-        public byte[] decrypt(char[] password, byte[] secret) throws GeneralSecurityException {
-            return getCipher(Cipher.DECRYPT_MODE, password, this.salt, this.iterationCount).doFinal(secret);
+        public Cipher getDecryptionCipher(char[] password) throws InvalidKeyException {
```

Comment:
![Codacy](https://www.codacy.com/assets/images/favicon.png) Issue found: [Consider using varargs for methods or constructors which take an array the last parameter.](https://www.codacy.com/app/ryotan/kaba-vault/file/2671050498/issues/source?bid=3298589&fileBranchId=3332567#l239)